README.md

```diff
@@ -37,6 +37,16 @@ $ ./matching_eval.py --predictions vsc_eval_data/matches.csv --ground_truth vsc_
 Matching track segment AP: 0.3650
 ```
 
+## Baseline results on training set
+
+| method                        | score norm         | Descriptor μAP | Matching μAP |
+|-------------------------------|--------------------|----------------|--------------|
+| [SSCD](docs/baseline.md)      | :x:                | 0.4754         | 0.3650       |
+| [SSCD](docs/baseline.md)      | :heavy_check_mark: | 0.6499         | 0.4692       |
+| [DINO](docs/baseline_dino.md) | :heavy_check_mark: | _0.4402_       | 0.3393       |
+| [DnS](docs/baseline_dns.md)   | :heavy_check_mark: | _0.4129_       | 0.3211       |
```

Comment:
Let's add a line below, something like "Numbers in _italics_ do not conform to challenge rules".

Reply:
Note added